### PR TITLE
New hospital model with serology

### PR DESCRIPTION
**Pull request by MJomaba**

### CODE REVIEW

**Comment by johnlees** [comment]
I've made a few changes to hopefully simplify a lot of the code in models.R for creating new models, and some similar changes in parameters.R. It was more tortured than it needed to be (my fault), hopefully this is clearer

Let me know when you're happy and I can do a review too

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: sircovid
 Title: SIR Model for COVID-19
-Version: 0.2.36
+Version: 0.2.37
```
Comment (johnlees) [inline DESCRIPTION:3]: Time for v0.3.0?

File: inst/extdata/fitted_parameters.csv [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-"gamma_hosp_R",0.109204638515258
-"s_triage",1
-"gamma_triage",0.509957901224795
-"s_ICU_D",2
-"gamma_ICU_D",0.291386107435204
-"s_ICU_R",2
-"gamma_ICU_R",0.354142862957898
-"s_stepdown",1
-"gamma_stepdown",0.452381010083587
+"","parameter","value"
```
Comment (johnlees) [inline inst/extdata/fitted_parameters.csv:1]: Why do we have the first column?

File: inst/extdata/severity_2020_04_12.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿,0 to 4,5 to 9,10 to 14,15 to 19,20 to 24,25 to 29,30 to 34,35 to 39,40 to 44,45 to 49,50 to 54,55 to 59,60 to 64,65 to 69,70 to 74,75 to 79,80 to 100
+,0 to 4,5 to 9,10 to 14,15 to 19,20 to 24,25 to 29,30 to 34,35 to 39,40 to 44,45 to 49,50 to 54,55 to 59,60 to 64,65 to 69,70 to 74,75 to 79,80 to 100
```
Comment (johnlees) [inline inst/extdata/severity_2020_04_12.csv:1]: What changed on these rows (I can't see the difference!)

File: R/odin.R [hunk 4]
```diff
@@ -25,6 +25,98 @@ basic <- function (beta_t, beta_y, D0, dt, E0, hosp_transmission, I0_asympt,
...
+hospital_with_serology <- function (beta_t, beta_y, comm_D_transmission, D0_comm, D0_hosp,
```
Comment (johnlees) [inline R/odin.R:29]: Is this one used?

**Comment by johnlees** [comment]
Also: consider changing the names of 'hospital_model' and the 'basic' model

File: inst/extdata/fitted_parameters.csv (hunk 2 above)
Comment (edknock) [reply inst/extdata/fitted_parameters.csv]: I've gotten rid of it, wasn't serving any purpose.

File: R/odin.R (hunk 4 above)
Comment (edknock) [reply R/odin.R]: Not anymore, have removed it now.

File: DESCRIPTION (hunk 1 above)
Comment (edknock) [reply DESCRIPTION]: Done

File: inst/extdata/severity_2020_04_12.csv (hunk 3 above)
Comment (edknock) [reply inst/extdata/severity_2020_04_12.csv]: Managed to fix this

**Pull request merged**